### DISCUSSION TITLE
电脑能翻墙，手机翻不了，换其他软件客户端能翻

### DISCUSSION THREAD

**Discussion by missbouo** [2dust/v2rayNG#3015] [General]
同一个节点，电脑能用，手机突然用不了，苹果和安卓都不能用，也同步时间了，没效果，换个其他翻墙软件，手机也能正常翻，这到底是怎么回事啊？求大佬告知，摆弄快一周了，还是不行

**Comment by sancdvs** [comment]
是不是因为用了最新版的xray的缘故

**Comment by missbouo** [comment]
不是的，刚开始低版本的不能用，我就下载最新版本的，还是不能用，我就郁闷哪里出问题了

**Comment by WeihanLi** [comment]
我也遇到了这个问题，我是小米手机，MIX 4, 感觉有点像是流量被系统拦截了，电脑上同样的配置是可以的

**Comment by Makitoid** [comment]
同样的问题，我用的是v2flyng，有些可以有些不可以

**Comment by ghost** [comment]
我是在xray启用了warp解锁流媒体后，手机端就一直连接不上。日志如下![Screenshot_2024-07-12-23-20-51-486_com.v2ray.ang.jpg](https://github.com/user-attachments/assets/28dec70d-6c38-4c4a-8f22-bc8b00c66f43)

**Comment by shiraayano** [comment]
这个问题我也遇到了，把内置dns打开后好了
